pkgs/top-level/python-packages.nix

```diff
@@ -2524,6 +2524,8 @@ in {
     doCheck = false; # lazy packager
   };
 
+  csscompressor = callPackage ../development/python-modules/csscompressor.nix;
```

Comment:
Ah, that's why nox-review said there was nothing to check --- I had assumed it was just because the actual derivation didn't change.